src/coprocessor/endpoint.rs

```diff
@@ -735,12 +735,12 @@ mod tests {
     impl RequestHandler for UnaryFixture {
         async fn handle_request(&mut self) -> Result<coppb::Response> {
             if self.yieldable {
-                // We split the task into small executions of 1 second.
-                for _ in 0..self.handle_duration_millis / 1_000 {
-                    thread::sleep(Duration::from_millis(1_000));
+                // We split the task into small executions of 1 millisecond.
```

Comment:
emm, in the code it seems to be split into executions of 100ms?

Reply:
yes...

Reply:
Updated